### ISSUE TITLE
Can't detect the game on Proton

### ISSUE THREAD

**Issue by Khrona98** [uberhalit/EldenRingFpsUnlockAndMore#61]
Hello there! I saw the other 2 closed issues about Linux/Proton and decided to try running the mod through that.

Unfortunately the mod can't detect and patch the game when run through Proton. Starting it before, after or alongside Elden Ring, the option is simply greyed out. Also trying to start the game with the dedicated button on the GUI just closes the app and the game isn't launched.

**Comment by uberhalit** [comment]
Hello, 
As of now, I do not intend of fixing this as I don't use Linux for gaming nor do I have any knowledge of Proton. If someone opens a PR I'll happily merge it.

**Comment by Khrona98** [comment]
Fair enough, hopefully someone with the knowledge required steps in and helps with this.

In the meantime it looks like it is possible to apply at least a few of these fixes on Linux by hex editing the game's executable. There is a [post about the FPS unlock on r/linux_gaming](https://www.reddit.com/r/linux_gaming/comments/t74ugn/increase_elden_ring_fps_cap_to_144_through_hex/) and I also [made one about the Ultrawide fix](https://www.reddit.com/r/linux_gaming/comments/t7v9jm/ultrawide_on_elden_ring_working_with_hex_edit/). I'd like to thank you for your work on this tool, both hex edits were made by looking at the GameData.cs file in this repo.

This is off-topic from the opened issue, but there's a fix for the animation rendering on ultrawide resolutions on FWS, but not on the FpsUnlockAndMore app. Do you know it this would also be possible to be done via hex editing?
